bindings/azure/cosmosdb/cosmosdb.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/a8m/documentdb"
 	"github.com/dapr/components-contrib/bindings"
 	"github.com/dapr/dapr/pkg/logger"
+	"github.com/yalp/jsonpath"
```

Comment:
Is there a way to do this without using this specific package? it looks unmaintained.

Reply:
In general, we want as little external dependencies if possible..

Reply:
That makes sense. I will programmatically parse the fields and navigate to the nested map.

Reply:
Done the changes. Please review.